main.go

```diff
@@ -15,6 +17,9 @@ var resources embed.FS
 func main() {
 	logger.PrintInfo("nearForm: k8s kurated addons CLI")
 
+	// load .env file
+	godotenv.Load(".env")
```

Comment:
It would be nice for the user to be able to specify the file, a `--config-file` flag would be more than enough

Reply:
If the configuration file doesn't exist it might be worth it to crash the program to avoid building things with the wrong parameters.